### PR TITLE
Docs release notes: June 28, 2024

### PR DESCRIPTION
**Pull request by paperclypse**
Docs release notes covering June 21-27, 2024

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+subject: "Docs"
+releaseDate: '2024-06-28'
+version: 'version: June 21-27, 2024'
+---
+
+### New docs
+
+* Mobile monitoring's [application not responding](/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding) helps you track and analyze when and why your Android application's blocked for more than five seconds.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:9]: > Mobile monitoring's [application not responding]

Feels like there's a noun like `feature` or `condition` that's missing. A possessive on `application not responding` feels weird

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* New browser monitoring [log](/docs/browser/new-relic-browser/browser-apis/log) and [wrapLogger](/docs/browser/new-relic-browser/browser-apis/wraplogger) API docs define the syntax, requirements, parameters, and examples on how to use these new endpoints.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:10]:
```suggestion
* New browser monitoring [`log()`](/docs/browser/new-relic-browser/browser-apis/log) and [`wrapLogger()`](/docs/browser/new-relic-browser/browser-apis/wraplogger) API docs define the syntax, requirements, parameters, and examples on how to use these new endpoints.
```

Method names should be wrapped in code with empty parens https://docs.newrelic.com/docs/style-guide/structure/code-examples/

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+### Major changes
+
+* We've removed our top-level `Other capabilities` category and migrated everything that was there into more appropriate categories. This should make it somewhat easier for you to find what you're looking for.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:14]: `somewhat easier` feels pretty wishy-washy. :) Let's sell it.

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* Updated [IAST troubleshooting](/docs/iast/troubleshooting/#see-my-application) with specific IP ranges to whitelist if you're not seeing your application.
+* Updated [Ruby agent configuration](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/#instrumentation-aws_sqs) with `NEW_RELIC_INSTRUMENTATION_AWS_SQS` auto-instrumentation configuration information.
+* Removed `ApplicationExitInfo` reporting configuration information from [configure mobile monitoring settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings), because you can do this in the UI.
+* [Workflow enrichments](/docs/alerts/get-notified/incident-workflows/#enrichments) support multiple variables so you can customize your data more specifically ot your needs.
+* Added the [MobileApplicationExit](/attribute-dictionary/?dataSource=Mobile&event=MobileApplicationExit) attribute to our data dictionary. Our data dictionary isn't only a docs resource, but it also feeds attribute definitions directly to the UI, for example when you mouseover chart elements.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:24]:
```suggestion
* Added the [`MobileApplicationExit`](/attribute-dictionary/?dataSource=Mobile&event=MobileApplicationExit) attribute to our data dictionary. Our data dictionary isn't only a docs resource, but it also feeds attribute definitions directly to the UI in the NRQL query builder.
```

Also, love the shoutout here for data dictionary flow. But, I don't think it works for chart elements—‚only in query builder AFAIK.

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    * Various other improvements
+* [Node.js agent v11.21.0](/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-11-21-0)
+    * Support for getting container IDs from ECS metadata API
+* [PHP agent v10.22.0.12](/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-22-0-12)
+    * EOL support for PHP 7.0 and 7.1
+    * Various bug fixes
+* [Ruby agent v9.11.0](/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-9-11-0)
+    * New instrumentation for the `aws-sdk-sqs` gem
+    * A couple of bug fixes
+
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:56]: Lotta empty lines at the end here

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+* Removed `ApplicationExitInfo` reporting configuration information from [configure mobile monitoring settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings), because you can do this in the UI.
+* [Workflow enrichments](/docs/alerts/get-notified/incident-workflows/#enrichments) support multiple variables so you can customize your data more specifically ot your needs.
+* Added the [MobileApplicationExit](/attribute-dictionary/?dataSource=Mobile&event=MobileApplicationExit) attribute to our data dictionary. Our data dictionary isn't only a docs resource, but it also feeds attribute definitions directly to the UI, for example when you mouseover chart elements.
+* Removed docs related to the outdated alerts REST API.
+* Our AI monitoring supports [NVIDIA NIM models](/docs/ai-monitoring/compatibility-requirements-ai-monitoring/#deploy-at-scale) and doesn't require any extra configuration outside of our manual or UI-based installation flows.
+
+### Release notes and What's New posts
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:28]: I like that you're now adding short summaries of release notes.

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+* Updated [Ruby agent configuration](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/#instrumentation-aws_sqs) with `NEW_RELIC_INSTRUMENTATION_AWS_SQS` auto-instrumentation configuration information.
+* Removed `ApplicationExitInfo` reporting configuration information from [configure mobile monitoring settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings), because you can do this in the UI.
+* [Workflow enrichments](/docs/alerts/get-notified/incident-workflows/#enrichments) support multiple variables so you can customize your data more specifically ot your needs.
+* Added the [MobileApplicationExit](/attribute-dictionary/?dataSource=Mobile&event=MobileApplicationExit) attribute to our data dictionary. Our data dictionary isn't only a docs resource, but it also feeds attribute definitions directly to the UI, for example when you mouseover chart elements.
+* Removed docs related to the outdated alerts REST API.
+* Our AI monitoring supports [NVIDIA NIM models](/docs/ai-monitoring/compatibility-requirements-ai-monitoring/#deploy-at-scale) and doesn't require any extra configuration outside of our manual or UI-based installation flows.
```
Comment (austin-schaefer) [inline src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:26]: This should go into major; this was a big headline announcement. Also consider linking to the blog post? https://newrelic.com/blog/how-to-relic/ai-monitoring-for-nvidia-nim

**Review by austin-schaefer — DISMISSED** [review]
Looks good, left a few minor comments and suggestions

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:9]: Good call

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:14]: Agreed!

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx (hunk 4 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:24]: Good to know!

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx (hunk 7 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:26]: Oh! OK!

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx (hunk 6 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:28]: Glad you like it!

File: src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx (hunk 5 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-6-28-2024.mdx:56]: I'll delete em

**Review by austin-schaefer — APPROVED** [review]

**Pull request merged**